### DISCUSSION TITLE
Время начала бигфармы

### DISCUSSION THREAD

**Discussion by TonitaN** [TonitaN/TFL2021#13] [Polls]
С учётом отмены лекции по распределенкам 24 декабря, остаётся только два нормальных варианта.

**Comment by mathhyyn** [comment]
Она будет 1 пару длиться?

**Reply by TonitaN** [reply]
Конечно же нет (иначе не успеете больше 3 задач, а то и 2). Будет длиться 2 пары, а после ещё скоростные грамматики и регулярки (чтобы никто не переживал, что решая на скорость, упустит часть остальной бигфармы).
